dockers/docker-database/Dockerfile.j2

```diff
@@ -12,6 +13,10 @@ RUN apt-get update
 # Install redis-server
 RUN apt-get install -y redis-tools redis-server
 
+RUN groupmod -g ${redis_user_gid} redis               && \
```

Comment:
>redis_user_gid [](http://example.com/codeflow?start=18&length=14)

Is there possibility that there is already other group occupied this gid?

Reply:
Fixed